app/services/domain/capacity-date-range.js

```diff
@@ -14,8 +18,31 @@ class CapacityDateRange {
       toYear
     ]
 
-    this.capacityFromDate = dateFormatter.build(fromDay, fromMonth, fromYear)
-    this.capacityToDate = dateFormatter.build(toDay, toMonth, toYear)
+    this.isValid()
+  }
+
+  isValid () {
+    var errors = ErrorHandler()
+
+    this.capacityFromDate = FieldSetValidator(this.fromFields, 'capacityFromDate', errors)
+      .isRequired()
+      .isValidDate()
+      .isPastDate()
+      .isLaterThan(moment().subtract(CASELOAD_CAPACITY.MAX_HISTORY, 'years'), 'maxCapacityHistory')
+      .getFormattedDate()
+
+    this.capacityToDate = FieldSetValidator(this.toFields, 'capacityToDate', errors)
+      .isRequired()
+      .isValidDate(this.capacityToDate)
+      .isPastDate(this.capacityToDate)
```

Comment:
Can the two date not be today's date?

Reply:
Good point, added `isPastOrPresentDate` to use instead.